reference/fleet/migrate-elastic-agent.md

```diff
@@ -224,27 +226,44 @@ This is best performed one policy at a time. For a given policy, you need to cap
     :screenshot:
     :::
 
-5. On the host machines where the current agents are installed, enroll the agents again using this copied URL and the enrollment token:
+5. Choose an approach:
 
-    ```shell
-    sudo elastic-agent enroll --url=<fleet server url> --enrollment-token=<token for the new policy>
-    ```
+    ::::{tab-set}
+    :::{tab-item} Fleet UI
 
-    The command output should be like the following:
+    {applies_to}`stack: ga 9.2` Migrate remote agents directly from the {{fleet}} UI:
 
-    :::{image} images/migrate-agent-install-command-output.png
-    :alt: Install command output
-    :screenshot:
+    1. In the source cluster, select the agents you want to migrate. Click the three dots next to the agents, and select **Migrate agents**.
+    2. In the migration dialog, provide the URI and enrollment token you obtained from the target cluster.
+    3. Use replace_token (Optional): When you are migrating a single agent, you can use the `replace_token` field to preserve the agent's original ID from the source cluster. This step helps with event matching, but will cause the migration to fail if the target cluster already has an agent with the same ID.
```

Comment:
This would just make it clearer that you're referencing a specific field:

```suggestion
    3. Use `replace_token` (Optional): When you are migrating a single agent, you can use the `replace_token` field to preserve the agent's original ID from the source cluster. This step helps with event matching, but will cause the migration to fail if the target cluster already has an agent with the same ID.
```